src/tests/simulation/src/cluster.rs

```diff
@@ -419,9 +419,9 @@ impl Cluster {
                 "--advertise-addr",
                 &format!("meta-{i}:5690"),
                 "--backend",
-                "etcd",
-                "--etcd-endpoints",
-                "etcd:2388",
+                "sql",
+                "--sql-endpoint",
+                "sqlite::memory:",
```

Comment:
Discussed it with @yezizp2012. Looks like currently each node has its own in-memory sqlite instance. They are not shared between nodes and not persisted after restart. This is not correct for simulation. We need a global shared sqlite instance that can be connected from all meta nodes. A possible solution is to add it as a global variable and enable it under `#[cfg(madsim)]`.

Reply:
Does this work?

https://arc.net/l/quote/gafyjdlk

Reply:
I suppose the connection string needs to be changed
```
rc = sqlite3_open("file::memory:?cache=shared", &db);
```

So that multiple instance can connect to the same db.